frontend/src/global_styles/layout/_main_menu.sass

```diff
@@ -67,13 +67,19 @@ $menu-item-line-height: 30px
       height: calc(100% - (var(--main-menu-item-height) + 10px)) // 10px spacing
       overflow: auto
       @include styled-scroll-bar
+    
+    .main-menu-wrapper
+      display: flex
+      flex-direction: column
+      height: 100%
 
   a:focus
     color: var(--main-menu-font-color)
       
   ul
     margin: 0
     padding: 0
+    overflow-y: auto
```

Comment:
@bsatarnejad Do you remember why this was added? I believe it creates the "Roadmap" issue from https://github.com/opf/openproject/pull/8753